### PR TITLE
[New Rule] Brute Force Detection - Windows

### PR DESCRIPTION
**Pull request by Samirbous**
https://github.com/elastic/detection-rules/issues/2164 (T1110 - Brute Force)

- multiple logon failure from same source address in 10s maxspan
- 5 logon failure followed by success from same source address in 5s maxspan
- 5 or more failed logon targeting a user.name with admin like naming conv from same src address

depend on https://www.elastic.co/guide/en/elasticsearch/reference/7.16/eql-syntax.html#eql-with-runs-statement added in 7.16

### CODE REVIEW

File: rules/windows/credential_access_bruteforce_admin_account.toml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+"""
+references = ["https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4625"]
+risk_score = 47
+rule_id = "f9790abf-bd0c-45f9-8b5f-d0b74015e029"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+sequence by source.ip with maxspan=10s
```
Comment (brokensound77) [inline rules/windows/credential_access_bruteforce_admin_account.toml:65]:
```suggestion
sequence by host.ip, source.ip with maxspan=10s
```

File: rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and
+malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the
+mean time to respond (MTTR).
+
+## Setup
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
```
Comment (brokensound77) [inline rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml:55]: Can you mention the relationship of this rule with 

name = "Multiple Logon Failure Followed by Logon Success"
rule_id = "4e85dc8a-3e41-40d8-bc28-91af7ac6cf60"


Anytime there are 10+ failures followed by success, both rules will trigger

File: rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+query = '''
+sequence by host.id, source.ip with maxspan=10s
+  [authentication where event.action == "logon-failed" and
+    /* event 4625 need to be logged */
+    winlog.logon.type : "Network" and
+    source.ip != null and source.ip != "127.0.0.1" and source.ip != "::1" and
+    not user.name : ("ANONYMOUS LOGON", "-", "*$") and not user.domain == "NT AUTHORITY" and
+
+    /* noisy failure status codes often associated to authentication misconfiguration */
+    not winlog.event_data.Status : ("0xC000015B", "0XC000005E", "0XC0000133", "0XC0000192")] with runs=10
```
Comment (brokensound77) [inline rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml:73]: nit:

could be worth explaining each code in a comment

**Review by brokensound77 — APPROVED** [review]
Thanks for including the guides - few comments then LGTM 👍

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**